etl.js

```diff
@@ -146,7 +145,40 @@ Etl.upsertChecklistItem = async function _upsertChecklistItem(item) {
     );
     projectMeta.amount = item._amount;
     store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
-    await sleep(SLEEP);
+  }
+
+  let secondaryAdminID = "5ff85abd2b962872d01fe3bf";
+  let fallbackOwnerField = card.customFieldItems.find(function (card) {
+    return card.idCustomField === secondaryAdminID;
+  });
+  let fallbackOwner = fallbackOwnerField?.value?.text;
+  let isExpectedFallbackOwner = fallbackOwner === projectMeta.fallback_owner;
+  if (fallbackOwner && !isExpectedFallbackOwner) {
+    await gh.projects.setFallbackOwner(
+      projectMeta.projectItemNodeId,
+      fallbackOwner
+    );
+    projectMeta.fallback_owner = fallbackOwner;
+    store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
+  }
+
+  let ownerArray = card.idMembers;
+  if (fallbackOwner) {
+    let fallbackOwnerId = Transform.usernameToId(fallbackOwner);
+  }
+  let ownerId = "";
+  if ((ownerArray.length = 1)) {
```

Comment:
I debated leaving this check in, because either way the filter will produce the correct outcome. this just saves a few cpu cycles I guess.